code/modules/mob/living/carbon/human/species/xenomorphs/alien_species.dm

```diff
@@ -110,9 +110,9 @@
 
 	return 0
 
-/datum/species/xenos/hug(var/mob/living/carbon/human/H,var/mob/living/target)
-	H.visible_message("<span class='notice'>[H] caresses [target] with its scythe-like arm.</span>", \
-					"<span class='notice'>You caress [target] with your scythe-like arm.</span>")
+/datum/species/xenos/tap(var/mob/living/carbon/human/H,var/mob/living/target)
```

Comment:
Would argue for keeping this. It's pretty neat in its spoop.

Reply:
Well I think we should keep both, but make tapping as default help click. And have hug as like emote or me.

Reply:
alright i'll make benos hug

Reply:
leaving the proc name like this tho but the text is a hug